stellar/stellar-anchor-server
# Adds webhook to record an external deposit.

This PR creates a webhook that can be used by the external entity to inform the anchor of a deposit. For now, this is extremely simple: `GET /deposit/confirm-transaction?transaction_id=XXXXX&amount=YYYY`. It closes #36 (part of #14, the deposit flow).

To test:
- Initiate a deposit using the `GET /deposit` endpoint; let `Y` be the deposit transaction ID.
- Input a valid amount `X` and reach the success page (`/deposit/success.html`)
- Using the above amount `X` and the deposit transaction ID `Y` as query parameters, call `GET deposit/confirm-transaction`
- Confirm that the transaction gets serialized with successful response code and status `pending_anchor`
